lib/services/db/backup/accounts.dart

```diff
@@ -92,12 +125,48 @@ class AccountsTable extends DBTable {
 
       if (queries != null) {
         for (final query in queries) {
-          await db.execute(query);
+          try {
+             if (query is String) {
+              await db.execute(query);
+            } else if (query is Function) {
+              await query(db);
+            }
+          } catch (e, s) {
+            debugPrint('$name migration error, index $i: $e');
+            debugPrintStack(stackTrace: s);
+          }
         }
       }
     }
   }
 
+    Future<void> _migrateProfiles(Database db) async {
```

Comment:
this migration is not needed. Every time an account is opened, we go to the wallet screen and go through the profile async loading flow which will load or create one